rep-2001.rst

```diff
@@ -38,26 +38,45 @@ We define three main entry points for ROS users.
  * desktop (recommended)
  * ros_base
  * ros_core
+ * ros_minimal
 
 Variants
 --------
 
+ROS Minimal
+'''''''''''
+
+The `ros_minimal` variant is a highly restricted set of ROS2 functionality.
+It targets resource- and tool-constrained environments where package managers may be limited or unavailable.
+The intention is that this variant can target cross-compiling for custom OS sysroots that may not have build tools or Python available that run on the target platform.
+It may not contain any GUI or Python dependencies.
+It may not have any dependencies outside a source workspace of its members with an exception of ubiquitous Linux libraries.
```

Comment:
Can you explicitly enumerate the dependencies of the current set of packages in the PR description?

Reply:
I was confusing this variant with another project that needed zlib and curl - the currently proposed package set requires no dependencies that the bare linaro sysroot does not include.